_posts/2016-06-30-servo-nightlies.md

```diff
@@ -0,0 +1,35 @@
+---
+layout:     post
+title:      Servo Nightly Builds Available
+date:       2016-06-30 20:00:00
+summary:    Try out the new Servo nightly builds
+categories:
+---
+
+Today the Servo team is making available nightly builds of the Servo web
```

Comment:
We are pleased to make nightly builds of the Servo web rendering engine available, starting today?